contracts/contracts/rem.system/src/voting.cpp

```diff
@@ -40,8 +40,16 @@ namespace eosiosystem {
 
       user_resources_table totals_tbl( _self, producer.value );
       const auto& tot = totals_tbl.get(producer.value, "producer must have resources");
+      const auto &voter = _voters.get(producer.value, "user has no resources");
+
       if ( prod != _producers.end() ) {
          if (!prod->active()) {
+             if(voter.locked_stake >= 250'000'0000ll){
```

Comment:
@AndriiAndrusyshyn it seems could be done easier; you can keep `_voters.modify` out of the braces but calculate lock time. e.g.
```
const auto lock_time = voter.locked_stake >= system_contract::producer_stake_threshold ? current_time_point()
                                                                                       : current_time_point() + _gstate.stake_lock_period;

_voters.modify(voter, producer, [&](auto &v) {
        v.stake_lock_time = lock_time
        v.locked_stake = tot.own_stake_amount;
    }
);
```